CONTRIBUTING.md

```diff
@@ -49,6 +49,9 @@ Use IntelliJ IDEA Community Edition/Ultimate.
    It can be significantly useful while working with UI components (or tests).
    To investigate the UI you may want to use `Tools > Internal Actions > UI > UI Inspector`.
 
+10. Consider setting `Class count before import with '*'` and `Names count to use static import with '*'` to a very high number (e.q. 500)
+to avoid problems with CheckStyle when editing code. To do that go to `Preferences > Code Style > Java/Kotlin/Scala`.
```

Comment:
Huh do you have `Preferences` in your IntelliJ, could you show the screenshot? For me it's sth like `File > Settings > Editor > Code Style > Java > Imports`

Reply:
<img width="904" alt="image" src="https://user-images.githubusercontent.com/61919032/188392368-02fc2a45-c275-43a2-8195-cb2021cdd97a.png">

"You can use ⌘, (macOS), or Ctrl+Alt+S (Windows/Linux) to display the Preferences/Settings dialog in IntelliJ IDEA."

[IntelliJ IDEA Guide](https://www.jetbrains.com/idea/guide/tutorials/configuring-intellij-idea/configuring-preferences-settings/)